EssentialFeed/EssentialFeediOSTests/CommentsSnapshotTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  CommentsSnapshotTests.swift
+//  EssentialFeediOSTests
+//
+//  Created by Robert Dates on 2/13/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeediOS
+@testable import EssentialFeed
+
+class CommentsSnapshotTests: XCTestCase {
+	func test_emptyComments() {
+		let sut = makeSUT()
+
+		sut.display(noComments())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "EMPTY_IMAGE_COMMENTS_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "EMPTY_IMAGE_COMMENTS_dark")
+	}
+	
+	func test_commentsWithErrorMessage() {
+		let sut = makeSUT()
+
+		sut.display(errorMessage("This is\na multi-line\nerror message"))
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "COMMENTS_WITH_ERROR_MESSAGE_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "COMMENTS_WITH_ERROR_MESSAGE_dark")
+	}
+	
+	func test_commentsWithContent() {
+		let sut = makeSUT()
+
+		sut.display(comments())
+
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .light)), named: "LOADED_COMMENTS_light")
+		assert(snapshot: sut.snapshot(for: .iPhone8(style: .dark)), named: "LOADED_COMMENTS_dark")
+	}
+	
+	// MARK: - Helpers
+	
+	private func makeSUT() -> CommentsViewController {
+		let bundle = Bundle(for: CommentsViewController.self)
+		let storyboard = UIStoryboard(name: "Comments", bundle: bundle)
+		let controller = storyboard.instantiateInitialViewController() as! CommentsViewController
+		controller.loadViewIfNeeded()
```

Comment:
Hide scroll indicators so they don't show up in the snapshots (check the snapshots to see it).

```suggestion
		let controller = storyboard.instantiateInitialViewController() as! CommentsViewController
		controller.tableView.showsVerticalScrollIndicator = false
		controller.tableView.showsHorizontalScrollIndicator = false
		controller.loadViewIfNeeded()
```

Reply:
done